toolkit/tools/imagegen/configuration/fileconfig.go

```diff
@@ -0,0 +1,119 @@
+// Copyright (c) Microsoft Corporation.
+// Licensed under the MIT License.
+
+//
+
+package configuration
+
+import (
+	"encoding/json"
+	"fmt"
+)
+
+// DestinationFileConfigList is a list of destination files to copy.
```

Comment:
```suggestion
// DestinationFileConfigList is a list of destination files to generate in the final image.
```

Reply:
Not critical, this just gets the idea across better to me.

Reply:
I changed it slightly.